server/api/region_test.go

```diff
@@ -258,6 +258,18 @@ func (s *testRegionSuite) TestTopSize(c *C) {
 	s.checkTopRegions(c, fmt.Sprintf("%s/regions/size?limit=%d", s.urlPrefix, 2), []uint64{7, 8})
 }
 
+func (s *testRegionSuite) TestAccelerateRegionsScheduleInRange(c *C) {
+	r1 := newTestRegionInfo(557, 13, []byte("a1"), []byte("a2"))
+	r2 := newTestRegionInfo(558, 14, []byte("a2"), []byte("a3"))
+	r3 := newTestRegionInfo(559, 15, []byte("a3"), []byte("a4"))
+	mustRegionHeartbeat(c, s.svr, r1)
+	mustRegionHeartbeat(c, s.svr, r2)
+	mustRegionHeartbeat(c, s.svr, r3)
+
+	err := postJSON(testDialClient, fmt.Sprintf("%s/regions/accelerate-schedule", s.urlPrefix), []byte(`{"start_key":"a1", "end_key": "a4"}`))
+	c.Assert(err, IsNil)
```

Comment:
I think testing with a range `{"start_key":"a1", "end_key": "a3"}` and checking if only two regions in SuspectRegions will be better?

Reply:
good idea, updated.